changelog.md

```diff
@@ -1,3 +1,29 @@
+# 0.2.1 Release
+This is the second release for the Ternoa Testnet.
+The two major features are the two new pallets capsule and associated-accounts.
+
+# 🚀 New Features
+The two major features are the two new pallets **capsule** and **associated-accounts** respectivelify for
+ - capsules management (creation/update/removal...).
+ - usernames management. 
+
+# Added
+- create extrinsic **(capsules)** 
```

Comment:
For the new pallets you can write
- new pallet: capsules - enables you to create a capsule out of a NFT
- new pallet: associated-accounts - enables you to link third party accounts with your main account

For new extrinsics:
- new marketplace extrinsic: set_uri - Allows you to store the link to your marketpklace

Reply:
✅